### PR TITLE
Fixing docs following breaking change

### PR DESCRIPTION
**Pull request by matbesancon**

### CODE REVIEW

**Comment by odow** [comment]
You can run with `doctests=:fix` to update the docs. We could do that in a separate PR.

File: test/Utilities/model.jl [hunk 1]
```diff
@@ -182,7 +182,7 @@ function test_ObjectiveFunction()
     return
 end
 
-function _test_T
+function _test_T()
```
Comment (odow) [inline test/Utilities/model.jl:185]: Haha. I didn't consider this edge case.

Comment (matbesancon) [reply test/Utilities/model.jl:185]: Was it caused by one of the regexes I assume?
